### PR TITLE
jnsgrukification

### PR DESCRIPTION
**Pull request by sed-i**

### CODE REVIEW

File: src/alertmanager_client.py [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+import logging
+import urllib.parse
+from typing import List, Optional
+
+import requests
+
+logger = logging.getLogger(__name__)
+
+
+class AlertmanagerAPIClient:
```
Comment (jnsgruk) [inline src/alertmanager_client.py:15]:
```suggestion
class Alertmanager:
```

File: src/alertmanager_client.py [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+    def reload(self) -> bool:
+        """Send a POST request to to hot-reload the config.
+        This reduces down-time compared to restarting the service.
+
+        Returns:
+          True if reload succeeded (returned 200 OK); False otherwise.
+        """
+        url = urllib.parse.urljoin(self.base_url, "/-/reload")
+        try:
+            response = requests.post(url, timeout=self.timeout)
```
Comment (jnsgruk) [inline src/alertmanager_client.py:31]:
```suggestion
            response = urllib.request.urlopen(url, data=None, timeout=self.timeout)
```

Perhaps drop the dependency on `requests`?

Comment (sed-i) [reply src/alertmanager_client.py:31]: Done

File: src/alertmanager_client.py (hunk 1 above)
Comment (sed-i) [reply src/alertmanager_client.py:15]: Done

**Review by jnsgruk — APPROVED** [review]
LGTM! :)

**Pull request merged**